spec/factories/list.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :list do
+    association :recipe, factory: :recipie
+    ingredient :ingredient, factory: :ingredient
+    amount         1.0
+    boil_time     0
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.